Description: Fix unversioned shared library build
Author: Kurt Kremitzki <kurt@kwk.systems>
Last-Updated: 2018-09-02

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -437,6 +437,13 @@
     set (SMESH_LIBS ${SMESH_LIBRARIES})
 endif(MSVC AND ENABLE_LIB_NAMING)
 
+if(BUILD_SHARED_LIBS)
+set_target_properties(${SMESH_LIBRARIES} PROPERTIES VERSION
+                      ${SMESH_VERSION_MAJOR}.${SMESH_VERSION_MINOR}.${SMESH_VERSION_PATCH}.${SMESH_VERSION_TWEAK})
+set_target_properties(${SMESH_LIBRARIES} PROPERTIES SOVERSION
+                      ${SMESH_VERSION_MAJOR}.${SMESH_VERSION_MINOR})
+endif()
+
 install(TARGETS ${SMESH_LIBRARIES}
         ARCHIVE DESTINATION "lib"
         RUNTIME DESTINATION "bin"
