Index: SMESH_SRC_6.5.0/src/Tools/padder/resources/padderexe/Makefile.am
===================================================================
--- SMESH_SRC_6.5.0.orig/src/Tools/padder/resources/padderexe/Makefile.am
+++ SMESH_SRC_6.5.0/src/Tools/padder/resources/padderexe/Makefile.am
@@ -24,11 +24,11 @@ mysalomeresdir=$(salomeresdir)/padderexe
 version=med3
 notinstall_version=med2
 
-# We install the padder.exe program and the files required for testing
-# the execution from within the installation directory
-dist_mysalomeres_SCRIPTS =    \
-	$(version)/padder.exe \
-	padder.sh
+#XXX## We install the padder.exe program and the files required for testing
+#XXX## the execution from within the installation directory
+#XXX#dist_mysalomeres_SCRIPTS =    \
+#XXX#	$(version)/padder.exe \
+#XXX#	padder.sh
 
 nodist_mysalomeres_SCRIPTS =         \
 	envPadder.sh
