Index: SMESH_SRC_6.5.0/idl/Makefile.am
===================================================================
--- SMESH_SRC_6.5.0.orig/idl/Makefile.am
+++ SMESH_SRC_6.5.0/idl/Makefile.am
@@ -79,6 +79,7 @@ OMNIORB_IDLCXXFLAGS = @OMNIORB_IDLCXXFLA
 OMNIORB_IDLPYFLAGS  = \
 	@OMNIORB_IDLPYFLAGS@ \
 	-I$(top_builddir)/idl/salome \
+	-I/usr/share/salome/idl/salome \
 	-I$(KERNEL_ROOT_DIR)/idl/salome \
 	-I$(MED_ROOT_DIR)/idl/salome \
 	-I$(GEOM_ROOT_DIR)/idl/salome
@@ -87,12 +88,14 @@ IDLCXXFLAGS = \
 	-bcxx \
 	@IDLCXXFLAGS@ \
 	-I$(top_builddir)/idl/salome \
+	-I/usr/share/salome/idl/salome \
 	-I$(KERNEL_ROOT_DIR)/idl/salome \
 	-I$(MED_ROOT_DIR)/idl/salome \
 	-I$(GEOM_ROOT_DIR)/idl/salome
 
 IDLPYFLAGS  = \
 	@IDLPYFLAGS@ \
+	-I/usr/share/salome/idl/salome \
 	-I$(KERNEL_ROOT_DIR)/idl/salome \
 	-I$(MED_ROOT_DIR)/idl/salome \
 	-I$(GEOM_ROOT_DIR)/idl/salome
@@ -132,7 +135,7 @@ mostlyclean-local:
 	@for dep in $^ dummy; do \
 	  if [ $$dep != "dummy" ]; then \
 	    echo Building dependencies for $$dep; \
-	    $(CPP) $(C_DEPEND_FLAG) -x c -I$(srcdir) -I$(KERNEL_ROOT_DIR)/idl/salome -I$(MED_ROOT_DIR)/idl/salome -I$(GEOM_ROOT_DIR)/idl/salome $$dep 2>/dev/null | \
+	    $(CPP) $(C_DEPEND_FLAG) -x c -I$(srcdir) -I/usr/share/salome/idl/salome -I$(KERNEL_ROOT_DIR)/idl/salome -I$(MED_ROOT_DIR)/idl/salome -I$(GEOM_ROOT_DIR)/idl/salome $$dep 2>/dev/null | \
 	    sed 's/\.o/\SK.cc/' >>$@; \
 	  fi; \
 	done ;
Index: SMESH_SRC_6.5.0/src/Tools/padder/meshjob/idl/Makefile.am
===================================================================
--- SMESH_SRC_6.5.0.orig/src/Tools/padder/meshjob/idl/Makefile.am
+++ SMESH_SRC_6.5.0/src/Tools/padder/meshjob/idl/Makefile.am
@@ -57,11 +57,13 @@ libSalomeIDLSPADDER_la_LIBADD     = \
 IDLCXXFLAGS = \
         -bcxx -I. \
 	@OMNIORB_IDLCXXFLAGS@ \
+	-I/usr/share/salome/idl/salome \
         -I$(KERNEL_ROOT_DIR)/idl/salome
 
 IDLPYFLAGS  = \
 	-I. \
 	@OMNIORB_IDLPYFLAGS@ \
+	-I/usr/share/salome/idl/salome \
         -I$(KERNEL_ROOT_DIR)/idl/salome
 
 ##########################################################
