diff --git a/src/SMDS/SMDS_MeshElementIDFactory.cxx b/src/SMDS/SMDS_MeshElementIDFactory.cxx
index 204d721..733a46c 100755
--- a/src/SMDS/SMDS_MeshElementIDFactory.cxx
+++ b/src/SMDS/SMDS_MeshElementIDFactory.cxx
@@ -93,7 +93,7 @@ bool SMDS_MeshElementIDFactory::BindID(int ID, SMDS_MeshElement * elem)
 //function : MeshElement
 //purpose  : 
 //=======================================================================
-SMDS_MeshElement* SMDS_MeshElementIDFactory::MeshElement(int ID)
+SMDS_MeshElement* SMDS_MeshElementIDFactory::MeshElement(int ID) const
 {
   if ((ID<1) || (ID>=myMesh->myCells.size()))
     return NULL;
diff --git a/src/SMDS/SMDS_MeshElementIDFactory.hxx b/src/SMDS/SMDS_MeshElementIDFactory.hxx
index 793ff0b..e1c5648 100755
--- a/src/SMDS/SMDS_MeshElementIDFactory.hxx
+++ b/src/SMDS/SMDS_MeshElementIDFactory.hxx
@@ -44,7 +44,7 @@ public:
   SMDS_MeshElementIDFactory();
   bool BindID(int ID, SMDS_MeshElement * elem);
   int SetInVtkGrid(SMDS_MeshElement * elem);
-  SMDS_MeshElement * MeshElement(int ID);
+  SMDS_MeshElement * MeshElement(int ID) const;
   virtual int GetFreeID();
   virtual void ReleaseID(int ID, int vtkId = -1);
   SMDS_ElemIteratorPtr elementsIterator() const;
diff --git a/src/SMDS/SMDS_MeshIDFactory.cxx b/src/SMDS/SMDS_MeshIDFactory.cxx
index ce8f45d..2a56a47 100755
--- a/src/SMDS/SMDS_MeshIDFactory.cxx
+++ b/src/SMDS/SMDS_MeshIDFactory.cxx
@@ -100,7 +100,7 @@ void SMDS_MeshIDFactory::SetMesh(SMDS_Mesh *mesh)
         myMesh = mesh;
 }
 
-SMDS_Mesh* SMDS_MeshIDFactory::GetMesh()
+SMDS_Mesh* SMDS_MeshIDFactory::GetMesh() const
 {
         return myMesh;
 }
diff --git a/src/SMDS/SMDS_MeshIDFactory.hxx b/src/SMDS/SMDS_MeshIDFactory.hxx
index c79e17a..a184409 100755
--- a/src/SMDS/SMDS_MeshIDFactory.hxx
+++ b/src/SMDS/SMDS_MeshIDFactory.hxx
@@ -42,7 +42,7 @@ public:
   virtual void Clear();
 
   void SetMesh(SMDS_Mesh *mesh);
-  SMDS_Mesh* GetMesh();
+  SMDS_Mesh* GetMesh() const;
   inline bool isPoolIdEmpty() { return myPoolOfID.empty(); };
   virtual void emptyPool(int maxId);
   inline void adjustMaxId(int ID) { if (ID > myMaxID) myMaxID = ID;};
diff --git a/src/SMESHUtils/SMESH_File.cxx b/src/SMESHUtils/SMESH_File.cxx
index b0341ba..64f46ae 100755
--- a/src/SMESHUtils/SMESH_File.cxx
+++ b/src/SMESHUtils/SMESH_File.cxx
@@ -37,6 +37,7 @@
 #include <io.h>
 #else
 #include <sys/mman.h>
+#include <unistd.h>
 #endif
 
 //================================================================================
diff --git a/src/SMESH_I/SMESH_Gen_i.hxx b/src/SMESH_I/SMESH_Gen_i.hxx
index 04edf2f..03008fc 100755
--- a/src/SMESH_I/SMESH_Gen_i.hxx
+++ b/src/SMESH_I/SMESH_Gen_i.hxx
@@ -425,7 +425,7 @@ public:
   // Copy-paste methods - returns true if object can be copied to the clipboard
   CORBA::Boolean CanCopy( SALOMEDS::SObject_ptr theObject ) { return false; }
   // Copy-paste methods - copy object to the clipboard
-  SALOMEDS::TMPFile* CopyFrom( SALOMEDS::SObject_ptr theObject, CORBA::Long& theObjectID ) { return false; }
+  SALOMEDS::TMPFile* CopyFrom( SALOMEDS::SObject_ptr theObject, CORBA::Long& theObjectID ) { return NULL; }
   // Copy-paste methods - returns true if object can be pasted from the clipboard
   CORBA::Boolean CanPaste( const char* theComponentName, CORBA::Long theObjectID ) { return false; }
   // Copy-paste methods - paste object from the clipboard
